### ISSUE TITLE
Authentication#emailPasswordlessStart returns 400 (Bad Request)

### ISSUE THREAD

**Issue by luckpoint** [auth0/auth0-PHP#668]
### SDK Version

8.3

### PHP Version

PHP 8.1

### What happened?

The following code gives 400 Bad Request error.
```php
$auth0 = new \Auth0\SDK\Auth0([
    'domain' => $_ENV['AUTH0_DOMAIN'],
    'clientId' => $_ENV['AUTH0_CLIENT_ID'],
    'clientSecret' => $_ENV['AUTH0_CLIENT_SECRET'],
    'cookieSecret' => $_ENV['AUTH0_COOKIE_SECRET']
]);

$authntication = $auth0->authentication();
$return = $authntication->emailPasswordlessStart("xxx@example.com", "link");
```

The following code works. 
```php
$auth0 = new \Auth0\SDK\Auth0([
    'domain' => $_ENV['AUTH0_DOMAIN'],
    'clientId' => $_ENV['AUTH0_CLIENT_ID'],
    'clientSecret' => $_ENV['AUTH0_CLIENT_SECRET'],
    'cookieSecret' => $_ENV['AUTH0_COOKIE_SECRET']
]);

$authntication = $auth0->authentication();
$return = $authntication->emailPasswordlessStart("xxx@example.com", "link", ["scope"=>"email openid"]);
```

emailPasswordlessStart() works when authParams explicitly is passed.

xxx@example.com should be replaced the actual email address.

### How can we reproduce this issue?

Just run the above code.

**Comment by evansims** [comment]
Hey @luckpoint 👋 Thanks for reporting this! I'll investigate and get back to you.

**Comment by evansims** [comment]
Hey @luckpoint 👋 Thanks again for your report, I've reproduced the issue and identified the cause. I'll get a fix released ASAP.

**Comment by evansims** [comment]
Hey again @luckpoint 👋 I've [released that fix as 8.3.7](https://github.com/auth0/auth0-PHP/releases/tag/8.3.7) — thanks!
